.github/workflows/rerun-workflows.yml

```diff
@@ -50,6 +50,35 @@ jobs:
               const { WebClient } = require('@slack/web-api');
               const slack = new WebClient('${{ secrets.SLACK_BOT_TOKEN }}');
 
+              // find out which jobs failed
+              const { owner, repo } = context.repo;
+
+              const jobInfo = await github.rest.actions.listJobsForWorkflowRun({
+                owner,
+                repo,
+                run_id: ${{ github.event.workflow_run.id }},
+                per_page: 100,
+              });
+
+              const failedJobs = jobInfo.data.jobs
+                .filter(job => job.status === 'completed' && job.conclusion === 'failure');
+
+              // don't bother slack if only the flaky job failed
+              if (failedJobs.length === 1 && failedJobs[0].name.includes("flaky")) {
+                return;
+              }
```

Comment:
Is it conceivable that multiple flaky jobs fail.

Reply:
I think we only have 1 flaky job now, right? This API call only gets the latest attempt, so we won't get duplicates across multiple attempts.

Reply:
you're right. this is nicer code and more future proof anyway

```ts
if (failedJobs.every(job => job.name.includes("flaky"))) {
  return;
}
```

Reply:
Oh, we're literally looking at the job name. Right. In that case it is only one.

Btw, wanted to propose `every`. Noice :)